features/aa/aa_features/orphans.feature

```diff
@@ -138,3 +138,21 @@ Feature:
     And I click the "Cancel" button
     Then I should be on the "Show Orphan" page for orphan "Orphan 1"
     And I should not see "Orphan N"
+
+  Scenario: **Bug fix OSRA385** Orphan filter should be done by name
```

Comment:
I'm not sure these thorough tests are necessary since the filter functionality is provided by the AA gem and not coded by us. You could probably dispense with tests altogether here and in sponsors.feature from #341.